weaviate/collections/classes/data.py

```diff
@@ -38,18 +39,36 @@ class DataObject(Generic[P, R]):
 
 
 @dataclass
-class DataReference:
-    """This class represents a reference between objects within a collection to be used when batching."""
-
+class _DataReference:
     from_property: str
     from_uuid: UUID
-    to_uuid: UUID
+    to_uuid: Union[UUID, List[UUID]]
+
+    def _to_uuids(self) -> List[UUID]:
+        if isinstance(self.to_uuid, uuid_package.UUID) or isinstance(self.to_uuid, str):
+            return [self.to_uuid]
+        else:
+            return self.to_uuid
 
 
 @dataclass
-class DataReferenceOneToMany:
```

Comment:
Can we leave this class in for BC and deprecate its usage with the intent to remove it for GA?

Reply:
I thought about it, but think it is not necessary:
- AFAIK it is not documented and there will be veeery few users
- it doesn't really fit into the current structure

(In general, I think we shouldn't overdo it with BC)